language/e2e_tests/src/account.rs

```diff
@@ -268,6 +271,7 @@ impl AccountData {
             MutVal::new(Value::U64(self.received_events_count)),
             MutVal::new(Value::U64(self.sent_events_count)),
             MutVal::new(Value::U64(self.sequence_number)),
+            MutVal::new(Value::Bool(self.delegated_withdrawal_capability)),
```

Comment:
This should be the second element in the vector right? Due to the alphabetical ordering of the compiler